DemoApp/src/Menu.h

```diff
@@ -10,6 +10,8 @@ class Menu : public AView {
     Menu(std::string title, std::vector<ViewPtr> children) : AView(std::move(title), std::move(children)) {}
 
     ViewPtr execute() override;
+    void printHelp() override;
+    bool executeCLI(const std::string &name, const std::map<std::string, std::string> &args) override;
 };
 
 #endif  // DEMOAPP_MENU_H
```

Comment:
newline :P

Reply:
Not my changes, it was'n there even before this PR. Its your code btw